filebeat/module/tidb/pd/config/baremetal.yml

```diff
@@ -0,0 +1,31 @@
+type: log
+
+paths:
+{{ range $i, $path := .paths }}
+ - {{$path}}
+{{ end }}
+
+exclude_files:
+  - '.gz$'
+
+processors:
+  - dissect:
+      tokenizer: '[%{time}] [%{tidb.pd.level}] [%{tidb.pd.source_line}] %{tidb.pd.log_message}'
```

Comment:
It is possible that not all log lines are conformed with the unified log format. For example, these may be logs occurred with existing unified log lines. Could you add test cases to confirm they work fine?

![image](https://user-images.githubusercontent.com/1916485/131857145-32b8728d-f035-4e2c-ad75-1bc46b20c883.png)

Reply:
Logs which are failed to parse will save as raw records, with an additional "parsing failed" field.

Reply:
This case should be supported after I improve processor pipeline.